semantic_text_search/semantic_text.ipynb

```diff
@@ -44,7 +44,7 @@
   },
```

Comment:
<p>Let's remove this summarize call, dimension:0 being returned is a bug</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pinecone-io/examples/pull/27/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>